staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```diff
@@ -227,7 +244,11 @@ func (o *ScaleOptions) RunScale() error {
 		}
 
 		mapping := info.ResourceMapping()
-		if err := o.scaler.Scale(info.Namespace, info.Name, uint(o.Replicas), precondition, retry, waitForReplicas, mapping.Resource); err != nil {
+		if o.dryRunStrategy == cmdutil.DryRunClient {
+			counter++
```

Comment:
remove this? :-)

Reply:
Moved the counter to the beginning of Visit since we use this to track items passed to `kubectl scale`:

https://github.com/kubernetes/kubernetes/blob/b7faac459fe9d7ff30e074465e3db2fd49daffeb/staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go#L254-L256